<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>250</width>
      <height>440</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="PauseMenu" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>98</width>
       <height>109</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { color: rgb(255, 255, 127) }</string>
     </property>
     <widget class="QPushButton" name="continueButton">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>48</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Continue</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exitPauseButton">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>77</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QLabel" name="pauseLabel">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>80</width>
        <height>33</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="menuWidget" native="true">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>160</y>
       <width>121</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { color: rgb(255, 255, 127) }</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="menuLabel">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Menu</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gameOverWidget" native="true">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>181</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { color: rgb(255, 255, 127) }</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="overLabel">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Game Over!!!</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="scoreOverTextLabel">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Score:</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="scoreOverLabel">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="restartButton">
        <property name="text">
         <string>Restart</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exitOverButton">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="helloMenu" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>231</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { color: rgb(255, 255, 127) }</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="helloNameLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel { color: rgb(255, 255, 127) }</string>
        </property>
        <property name="text">
         <string>F1 Race</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="helloModeLabel">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Choose the game 
mode</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="startClassicButton">
        <property name="text">
         <string>Classic mode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startTripButton">
        <property name="text">
         <string>Trip mode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="helloExitButton">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <zorder>menuWidget</zorder>
    <zorder>PauseMenu</zorder>
    <zorder>menuWidget</zorder>
    <zorder>gameOverWidget</zorder>
    <zorder>helloMenu</zorder>
   </widget>
   <widget class="QLabel" name="ScoreTextLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Score:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ScoreLabel">
    <property name="geometry">
     <rect>
      <x>276</x>
      <y>110</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="SpeedTextLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Speed:</string>
    </property>
   </widget>
   <widget class="QLabel" name="SpeedLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="exitToMenuButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>320</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Exit to main menu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>360</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Quit the game</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>menuWidget</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>continueButton</sender>
   <signal>clicked()</signal>
   <receiver>PauseMenu</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitPauseButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>438</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitOverButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>helloExitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startClassicButton</sender>
   <signal>clicked()</signal>
   <receiver>helloMenu</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startTripButton</sender>
   <signal>clicked()</signal>
   <receiver>helloMenu</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitToMenuButton</sender>
   <signal>clicked()</signal>
   <receiver>helloMenu</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>390</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
